core/drivers/crypto/hisilicon/sub.mk

```diff
@@ -8,3 +8,4 @@ srcs-$(CFG_HISILICON_ACC_V3) += hpre_main.c
 srcs-$(CFG_HISILICON_ACC_V3) += hpre_dh.c
 srcs-$(CFG_HISILICON_ACC_V3) += hpre_ecc.c
 srcs-$(CFG_HISILICON_ACC_V3) += hpre_montgomery.c
+srcs-$(CFG_HISILICON_ACC_V3) += hpre_rsa.c
```

Comment:
Please add the missing newline at the end of the file.

Reply:
solved